near-sdk/src/store/tree_map/entry.rs

```diff
@@ -135,6 +135,7 @@ where
     /// assert_eq!(map["poneyland"], None);
     /// # }
     /// ```
+    #[allow(clippy::unwrap_or_default)]
```

Comment:
Let's move it to the actual line of code instead of applying to the whole function